packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/platform/SystemUtils.kt

```diff
@@ -109,6 +110,11 @@ public expect fun threadId(): ULong
  */
 public expect fun epochInSeconds(): Long
 
+/**
+ * Returns a RealmInstant representing the time that has passed since the Unix epoch.
+ */
+public expect fun internalNow(): RealmInstant
```

Comment:
We only make this `public` if there is a reason to. All access to this seems to happen inside the `library-base` module, so it should be safe to mark this `internal`